src/main/java/fr/uge/chargepointconfiguration/chargepoint/ChargePointManager.java

```diff
@@ -40,22 +40,17 @@ public ChargePointManager(OcppVersion ocppVersion,
    *
    * @param webSocketRequestMessage WebSocketRequestMessage.
    */
-  public void processMessage(WebSocketRequestMessage webSocketRequestMessage) {
+  public String processMessage(WebSocketRequestMessage webSocketRequestMessage) {
     try {
       var message = ocppMessageParser.parseMessage(webSocketRequestMessage);
-      userRepository.save(new User("Borne",
-              "toBeALive",
-              webSocketRequestMessage.messageName(),
-              "g00d p4ssw0rd",
-              User.Role.Visualizer));
-      var resp = new BootNotificationResponse(LocalDateTime.now().toString(),
-              60,
-              RegistrationStatus.Accepted);
+      var resp = ocppMessageBuilder.buildMessage(webSocketRequestMessage);
       messageSender.sendMessage(new WebSocketResponseMessage(3,
               webSocketRequestMessage.messageId(),
               JsonParser.objectToJsonString(resp)));
+      return JsonParser.objectToJsonString(resp);
     } catch (IllegalArgumentException e) {
       System.out.println(e.getMessage());
+      return "ERROR";
```

Comment:
what about that, what do we get from this return ?

Reply:
We should ask @Mamar-dev what to do.
We could throw an exception or maybe silently disconnect the machine ?

Reply:
Removed the IllegalArgumentException, I don't know why is it used !

See commit [c382bd1](https://github.com/LastProject-ESIEE/OCPPConfigurationServer/pull/24/commits/c382bd1766b2e4fd1acf152f10bb4721ea487f13)